host/sgx/sgxquoteprovider.c

```diff
@@ -337,3 +337,237 @@ void oe_free_qe_identity_info_args(oe_get_qe_identity_info_args_t* args)
             free(args->host_out_buffer);
     }
 }
+
+oe_result_t oe_get_quote_verification_collateral(
+    oe_get_quote_verification_collateral_args_t* args)
+{
+    oe_result_t result = OE_FAILURE;
+    sgx_plat_error_t r = SGX_PLAT_ERROR_OUT_OF_MEMORY;
+    sgx_ql_qve_collateral_t* collateral = NULL;
+    uint32_t host_buffer_size = 0;
+    uint8_t* p = 0;
+    uint8_t* p_end = 0;
+    OE_TRACE_INFO("Calling %s\n", __FUNCTION__);
+
+    uint8_t* fmspc = args->fmspc;
+    uint16_t fmspc_size = sizeof(args->fmspc);
+    uint8_t* pck_ca = args->pck_ca;
+
+    OE_CHECK(oe_initialize_quote_provider());
+
+    if (!provider.get_quote_verification_collateral ||
+        !provider.free_quote_verification_collateral)
+    {
+        OE_TRACE_WARNING("Warning: Quote verification collateral was not "
+                         "supported by quote provider\n");
+        result = OE_QUOTE_PROVIDER_CALL_ERROR;
+        goto done;
+    }
+
+    // fetch collateral information
+    r = provider.get_quote_verification_collateral(
+        fmspc, fmspc_size, (char*)pck_ca, &collateral);
+    if (r != SGX_PLAT_ERROR_OK || collateral == NULL)
+    {
+        OE_RAISE(OE_QUOTE_PROVIDER_CALL_ERROR);
+    }
+
+    if (collateral->version != 1)
```

Comment:
SGX_QL_QVE_COLLATERAL_VERSION?

Reply:
Thanks, changed to SGX_QL_QVE_COLLATERAL_VERSION